### PR TITLE
[WIP] Release Candidate

### PR DESCRIPTION
**Pull request by randallli**
#### Don't forget:
- [ ] Identify the component the PR relates to in brackets in the title. ```[Buttons] Updated documentation```
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [ ] Sign the CLA bot. You can do this once the pull request is opened.

[Contributing](./contributing/README.md#pull-requests) has more information and tips for a great
pull request.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,141 @@
...
+                                                           emphasis:MDCActionEmphasisLow
+                                                            handler:handler];
+  [alert addAction:disagreeAaction];
+  [MDCAlertControllerThemer applyScheme:self.alertScheme toAlertController:alert];
+
+  [self presentViewController:alert animated:YES completion:NULL];
+```
+## Component changes
+
+### MDCFloatingButtonThemer changes its mapping to use a title color [#5912](https://github.com/material-components/material-components-ios/commit/df96ba447e2ace5554eb3acdfb483cae57ddc21e)
```
Comment (jverkoey) [inline CHANGELOG.md:57]:
```suggestion
### Buttons

MDCFloatingButtonThemer changes its mapping to use a title color [#5912](https://github.com/material-components/material-components-ios/commit/df96ba447e2ace5554eb3acdfb483cae57ddc21e)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,141 @@
...
+
+### Banner
+
+A MVP view with layout was added to examples.
```
Comment (jverkoey) [inline CHANGELOG.md:61]:
```suggestion
An MVP view with layout was added to examples.
```

**Review by jverkoey — APPROVED** [review]

**Comment by jverkoey** [comment]
```
ERROR: /Volumes/BuildData/tmpfs/src/github/repo/components/ButtonBar/BUILD:111:1: Compiling Swift module unit_test_swift_sources (10 files) failed (Exit 1)
components/ButtonBar/tests/unit/ButtonBarThemingTests.swift:21:8: error: no such module 'components_ContainerScheme_ContainerScheme'
import components_ContainerScheme_ContainerScheme // Alpha
       ^
```

File: CHANGELOG.md (hunk 1 above)
Comment (randallli) [reply CHANGELOG.md:57]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (randallli) [reply CHANGELOG.md:61]: Done.

**Comment by randallli** [comment]
OK CLA i checked the commits and authors and they all signed the CLA

**Pull request merged**